app/src/test/kotlin/net/corda/cli/application/commands/SetCurrentNodeCommandTest.kt

```diff
@@ -1,42 +1,42 @@
 package net.corda.cli.application.commands
 
-import com.github.stefanbirkner.systemlambda.SystemLambda.withEnvironmentVariable
-import net.corda.cli.application.App
-import net.corda.cli.application.utils.Files
-import org.junit.jupiter.api.Assertions.assertEquals
-import org.junit.jupiter.api.Test
-import org.yaml.snakeyaml.Yaml
-import picocli.CommandLine
-import java.io.FileInputStream
-import java.io.FileWriter
-import java.util.*
+// TODO: https://r3-cev.atlassian.net/browse/CORE-15885
```

Comment:
Agreed with @josephzunigadaly that this can be resolved later so it doesn't block the J17 upgrade